src/actionbar/index.css

```diff
@@ -0,0 +1,53 @@
+@import '../commons/index.css';
+
+:root {
+  --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
+  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-margin-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-offset-y: var(--spectrum-global-dimension-size-200);
+
+  --spectrum-actionbar-min-width: 280px;
+  --spectrum-actionbar-max-width: 1280px;
+}
+
+/* topdoc
+{{ actionbar/actionbar.yml }}
+*/
+.spectrum-ActionBar {
+  align-items: center;
+  flex-direction: initial;
+  justify-content: space-between;
+
+  height: var(--spectrum-actionbar-height);
+  padding: 0 var(--spectrum-actionbar-padding-x);
+  min-width: var(--spectrum-actionbar-min-width);
+  max-width: var(--spectrum-actionbar-max-width);
+  width: 100%;
+}
+
+/* topdoc
+{{ actionbar/table-actionbar.yml }}
+*/
+.spectrum-ActionBar-container {
+  margin: 0 var(--spectrum-actionbar-margin-x);
```

Comment:
An example would probably make this clear, but I'm not sure what this is intended to do, and I wonder if it should have been padding?

Reply:
This margin setting comes from here
![image](https://user-images.githubusercontent.com/1207520/57488451-f9397100-7267-11e9-9dea-c8eb0cce764a.png)

Reply:
Ok, looks like it needs be padding on `.spectrum-ActionBar`

Reply:
I tried it [here](https://codepen.io/jianliao/pen/ROmMdj), does not seem right.

Reply:
Hmm ok, my concern is that margin will have no effect. But if it's working correctly with margin, then I'm fine with it.